lib/msal-browser/src/config/Configuration.ts

```diff
@@ -38,6 +38,7 @@ export type BrowserAuthOptions = {
     navigateToLoginRequestUrl?: boolean;
     clientCapabilities?: Array<string>;
     protocolMode?: ProtocolMode;
+    azureCloudInstance?: AzureCloudInstance;
```

Comment:
We should update the in-file documentation of this type too, to include `azureCloudInstance`

Reply:
Yep. On it.